README.md

```diff
@@ -149,3 +149,21 @@ If you're interested, you can see the code for the News Summary API in this repo
 * [Aylien text summary API docs](http://docs.aylien.com/docs/summarize)
 * cURL [man page](https://curl.haxx.se/docs/manpage.html)
 * [Hurl](https://www.hurl.it/), a web interface for sending HTTP requests
+
+# My Solution
+
+## My Approach
+* Used <a href="https://open-platform.theguardian.com/explore/">Guardian Open Platform</a> to explore the API data.
+* `fetch` to access and manipulate API requests and responses.
+* Document Hashes - to retain single page.
+* Visualising DOM Tree Elements on paper (no diagram).
+
+
+## TODOs
+* when using Makers News Summary API, thumbnail isn't retrieved? - I stubbed it instead (imagine that I had it). Should sign up for a dev account and use a private API key.
+* functions that deal with data from Aylien API should probably be separated into its own class.
+* Improve structure of interface
+* Git good at vanilla JS 🙈.
+* Refactoring, more testing.
+* The CSS is an abomination...
+
```

Comment:
I really like your README, mentioning the approach you made and also what you need to work on is very good.